arroyo/utils/metrics.py

```diff
@@ -113,11 +116,12 @@ def decrement(self, value: float = 1.0) -> None:
 
 def configure_metrics(metrics: Metrics) -> None:
```

Comment:
can we add a function argument here that allows you to opt into overriding? In sentry we have caught quite a few initialization bugs by arroyo raising because metrics were configured twice.

```
def configure_metrics(metrics: Metircs, override=False): ...
```

Reply:
Good idea, updated